src/Accelerators/NNPA/Compiler/NNPACompilerUtils.cpp

```diff
@@ -49,11 +49,56 @@ using namespace onnx_mlir;
 namespace onnx_mlir {
 
 void configurePassesNNPA() {
-  configureOnnxToZHighLoweringPass(optReport == OptReport::NNPAUnsupportedOps);
   // z16 does not support for hardware saturation.
   // So, force its usage to compiler generated sticks.
   if (nnpaEnableSaturation && isLessEqualNNPALevel(NNPALevel::M14))
     nnpaEnableCompilerStickUnstick = true;
+
+  // Configure ONNXToZHighLoweringPass.
+  bool isDynQuant = !nnpaQuantDynamic.empty();
+  bool isWeightSym = false;
+  bool isActivationSym = false;
```

Comment:
quick question: why is the original init false/false here, but the auto value below is true/false.

Should't the default with no values being the same as the auto? Is there a reason that we want the "no options whatsoever" be different than auto?

Reply:
Yes, it should be the same. Fixed it. Thanks!